### PR TITLE
fix: Prevent a couple warnings from being displayed in stdout

### LINKED ISSUE #9048: 5.2.0 Regression: error message printing saying Couldn't find tsconfig.json
### Current behavior
With version 5.2.0, 5.3.0, 5.4.0, 5.5.0
Using a `tsconfig.json` in the cypress directory.
Run the command `cypress open`
The output displays:

```
Couldn't find tsconfig.json. tsconfig-paths will be skipped
```

### Desired behavior
Use the `tsconfig.json` : it works with version 5.1.0

### Test code to reproduce
`tsconfig.json` in `{root_project _directory}/cypress/`

```js
{
	"compilerOptions": {
		"strict": true,
		"sourceMap": false,
		"declaration": false,
		"module": "esnext",
		"moduleResolution": "node",
		"emitDecoratorMetadata": true,
		"esModuleInterop": false,
		"experimentalDecorators": true,
		"importHelpers": true,
		"resolveJsonModule": true,
		"allowSyntheticDefaultImports": true,
		"target": "es6",
		"lib": ["es6", "dom"],
		"types": ["cypress", "node"]
	},
	"include": ["**/*.ts"]
}
```

### Versions
5.2.0, 5.3.0, 5.4.0, 5.5.0
Work with 5.1.0

### PR DESCRIPTION
**Pull request by chrisbreiding**
- Closes #9048
- Addresses comments in https://github.com/cypress-io/cypress/issues/2522#issuecomment-767576294
- Addresses [TR-463](https://cypress-io.atlassian.net/browse/TR-463)

### User facing changelog

- There will no longer be a warning displayed when the video cannot be found after a run.
- There will no longer be a warning displayed when running a TypeScript spec without a tsconfig.json file present.

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue or this PR been tagged with a release in ZenHub? 
- N/A Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- N/A Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- N/A Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: patches/tsconfig-paths+3.9.0.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
++++ b/node_modules/tsconfig-paths/lib/register.js
+@@ -51,7 +51,7 @@ function register(explicitParams) {
+         explicitParams: explicitParams
+     });
+     if (configLoaderResult.resultType === "failed") {
+-        console.warn(configLoaderResult.message + ". tsconfig-paths will be skipped");
++        // console.warn(configLoaderResult.message + ". tsconfig-paths will be skipped");
+         return noOp;
+     }
+     var matchPath = match_path_sync_1.createMatchPath(configLoaderResult.absoluteBaseUrl, configLoaderResult.paths, configLoaderResult.mainFields, configLoaderResult.addMatchAll);
```
Comment (flotwig) [inline patches/tsconfig-paths+3.9.0.patch:13]: i think this belongs in `packages/server`, no?

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "effective:circle:config": "circleci config process circle.yml | sed /^#/d",
     "ensure-deps": "./scripts/ensure-dependencies.sh",
     "get-next-version": "node scripts/get-next-version.js",
-    "postinstall": "./scripts/run-if-not-ci.sh yarn build",
+    "postinstall": "patch-package && ./scripts/run-if-not-ci.sh yarn build",
```
Comment (flotwig) [inline package.json:37]: `packages/server`?

**Review by flotwig — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (flotwig) [reply package.json:37]: might need a nohoist

**Review by flotwig — APPROVED** [review]

**Review by NeverLiez — COMMENTED** [review]
-

**Pull request merged**